docs/software/cloud.md

```diff
@@ -192,9 +193,10 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
         bantime = 600
         ```
 
-        Assure that the `logpath` matches the `'datadirectory'` value in the Nextcloud configuration file (`config.php` see above), or the `'logfile'` value if defined.
+        Assure that the `logpath` matches the `'datadirectory'` value in the Nextcloud configuration file (`config.php`, see above), or the `'logfile'` value, if defined.
```

Comment:
The last comma is indeed wrong there, at least optional. And if used optional, it makes the meaning unclear, since it refers to the `logfile` only, as the `datadirectory` is always defined.